webrtc-stats.html

```diff
@@ -2122,12 +2122,12 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  It is the current target bitrate configured for this particular <a>SSRC</a> and is the
-                  Transport Independent Application Specific (TIAS) bitrate [[!RFC3890]].
-                  Typically, the target bitrate is a configuration parameter provided to the
-                  codec's encoder and does not count the size of the IP or other transport layers
-                  like TCP or UDP. It is measured in bits per second and the bitrate is calculated
-                  over a 1 second window.
+                  Reflects the current encoder target in bits per second. The
+                  target is an instantanous value reflecting the encoder's
+                  settings, but the resulting payload bytes sent per second,
+                  excluding retransmissions, SHOULD closely correlate to the
+                  target. See also {{RTCSentRtpStreamStats/bytesSent}} and
+                  {{RTCOutboundRtpStreamStats/retransmittedBytesSent}}.
```

Comment:
The targetBitrate is defined in the same way as the Transport Independent Application Specific (TIAS) bitrate [[!RFC3890]].